### PR TITLE
Add longest side data & adjust stats of glass items

### PR DESCRIPTION
**Pull request by DialoMalison**
#### Summary

Summary: None

#### Purpose of change

Add longest side data to glass items and adjust stats to better reflect reality.

#### Describe the solution
Add the data.

#### Describe alternatives you've considered

Not doing it.

### CODE REVIEW

File: data/json/items/containers.json [hunk 1]
```diff
@@ -898,7 +900,8 @@
     "looks_like": "flask_glass",
     "description": "A 10ml laboratory cylindrical test tube, with a rubber stopper.",
     "weight": "36 g",
-    "volume": "11ml",
+    "volume": "92 ml",
```
Comment (meladath) [inline data/json/items/containers.json:903]: A 10ml flask is 92ml in volume? Surely the glass itself isnt 82ml? I think I am missing something.

Comment (DialoMalison) [reply data/json/items/containers.json:903]: You're not missing anything. A 10 ml test tube is 100 mm x 13 mm. The volume of a cylinder of those dimensions is 13.2732 cm3 or aprox. 13 ml. God knows what numbers I entered in my calculator to get that result. 

Good catch, thanks.

File: data/json/items/containers.json [hunk 2]
```diff
@@ -256,11 +256,11 @@
     "looks_like": "bottle_plastic",
     "description": "A resealable glass bottle, holds 750 ml of liquid.",
     "weight": "450 g",
-    "volume": "1734 ml",
+    "volume": "1318 ml",
     "longest_side": "276 mm",
     "price": 0,
     "price_postapoc": 10,
-    "to_hit": 1,
+    "to_hit": 1,Fglass
```
Comment (Wishbringer) [inline data/json/items/containers.json:263]: Accidental paste?

Comment (DialoMalison) [reply data/json/items/containers.json]: Failed search (shift+f instead of ctrl+f). Thanks for catching it.

**Comment by DialoMalison** [comment]
OK, I got one (badly) wrong, so I re-ran the calculations on the others. Yep, they were all too high. I should have slept and then ran the calculations instead of the other way around. I think all the values are correct now.

**Pull request merged**